pkg/event/queue_test.go

```diff
@@ -22,14 +22,25 @@ import (
 	"testing"
 )
 
+func TestInMemoryQueue_Add(t *testing.T) {
```

Comment:
nit. This could probably be more descriptive i.e. `TestInMemoryQueue_Add_Size_Bounded_By_MaxSize`?